roadmap/roadmap.md

```diff
@@ -0,0 +1,90 @@
+# The Astropy Roadmap
+
+- [Status Legend](#Status-Legend)
+- [Functionality](#Functionality)
+- [Hardware and Performance](#Hardware-and-Performance)
+- [Learn and User Support](#Learn-and-User-Support)
+- [Community Building and Sustainability](#Community-Building-and-Sustainability)
+- [Infrastructure, Documentation](#Infrastructure-Documentation)
+- [Government, Management, and Personnel](#government-management-and-personnel)
+
+
+## Status Legend
+
+### Green :green_circle:
+- Group or person leading effort is identified and sufficiently supported.
+- Effort is already underway and/or expected to be implemented in the “near term.”
+
+### Orange :large_orange_diamond:    
+- Item is already defined (e.g., issue, APE exists), but not sufficiently underway.

```

Comment:
How is “sufficiently” defined? For example, if somebody has started working on an item, but somebody else thinks the progress they are making is too slow, is it “sufficiently underway”?

Reply:
This should probably be left vague and determined on a case-by-case basis. Stale timer might be different for a large complex item compared to a simpler one... 💭